WebApi/Controllers/BotBasicController.cs

```diff
@@ -21,7 +21,22 @@ public BotBasicController(ITelegramCommunicationService telegramCommunicationSer
         [HttpPost("update")]
         public async Task<IActionResult> Update([FromBody] object update)
         {
-            await _telegramCommunicationService.GetMessage(update);
+
+            var upd = JsonConvert.DeserializeObject<Update>(update.ToString());
+
+            if (upd?.Message?.Chat == null)
+            {
+                return Ok();
```

Comment:
BadRequest("Object is null, or Message is null, or Chat is null");

Reply:
Done